templates/video.hbs

```diff
@@ -0,0 +1,26 @@
+<section class="container">
+
+    <div class="info-video__section">
+
+        <div class="paragraph">
+            <div class="paragraph_title">
+                <p>{{titleText}}</p>
+            </div>
+            <div>
+                <p class="paragraph_text">{{paragraphText}}
+                </p>
+            </div>
+        </div>
+
+        <div class="video-section"> 
+            <video  controls> 
+                <source src={{assetSrc videoInfo}}  type="video/mp4">
+                <source src="/asset/video.ogg"  type="video/ogg">
+                <source src="/asset/video.webm"  type="video/wav">
```

Comment:
remove the source elements of .ogg and .webm formats as we don't have those videos...